### DISCUSSION TITLE
Auto Partitioning

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#7866] [Unsupported Versions] [answered]
Hi,
I’ve read here that Security Onion 2.3 partitions itself automatically when using the iso
[https://docs.securityonion.net/en/2.3/partitioning.html](https://docs.securityonion.net/en/2.3/partitioning.html) 

How will that work when adding more resources to the VM (adding storage) after the basic installation?

If it doesn't handle that automatically how am I suppose to make it work?

**Comment by tomjones1977** [comment, marked as answer]
You should be able to use standard LVM commands to expand the logical volume you want to change.  For example you can expand the nsm volume if you want to store more pcap/DB data.

There are other ways to increase available space, especially if the device is a VM as you can add more virtual drives and mount nsm to them.  https://docs.securityonion.net/en/2.3/new-disk.html
